### ISSUE TITLE
Enable git-lfs installing in orb

### ISSUE THREAD

**Issue by Cody-G** [elementary-robotics/atom#239]
crossref https://github.com/elementary-robotics/element-barcode/pull/7#discussion_r371334802

It would be great to alter the Atom orb to make it easier to install and use git-lfs.  Below are the steps I took to install git-lfs in the CircleCI config of element-barcode, which also required setting `executor: atom/build-ubuntu`.

cc @aywaldron 

```
        - run:                                                                                                          
            name: Install Git Large File Storage (LFS)                                                                  
            command: |                                                                                                  
              curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | sudo bash             
              sudo apt-get install -f git-lfs && git lfs install                                                        
              mkdir -p ~/.ssh/ && echo -e "Host github.com\n\tStrictHostKeyChecking no\n" > ~/.ssh/config               
              ssh-keyscan -H github.com >> ~/.ssh/known_hosts                                                           
              ssh git@github.com git-lfs-authenticate "${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}" download  
```

**Comment by aywaldron** [comment]
@Cody-G @dpipemazo A development orb tagged `elementaryrobotics/atom@dev:239` has been published to test the addition of this command. Since we recently merged in other changes to the orb, testing with this dev orb can also test that those changes are not breaking existing builds. Let me know if you want me to try it out on `element-barcode`.

**Comment by Cody-G** [comment]
>Let me know if you want me to try it out on element-barcode

That would be great!  So after your changes can we eliminate all of [these](https://github.com/elementary-robotics/element-barcode/blob/master/.circleci/config.yml#L19-L26) lines?  Or we still need to authenticate?

**Comment by aywaldron** [comment]
@Cody-G I included the authentication in the install command so yes. We can also split them out into separate commands if we want the flexibility to not have to run installation and authentication at the same time.

**Comment by Cody-G** [comment]
I don't see a problem with running installation and authentication at the same time, as long as the authentication in the orb is run again again at the time of the job (don't know enough about how orbs work to answer that myself)

**Comment by aywaldron** [comment]
It will be run on every job. 
I created a ticket at https://github.com/elementary-robotics/element-barcode/issues/10 for updating the `element-barcode` build.

**Comment by Cody-G** [comment]
Woohoo, closed by #242
